content/the-ember-times-issue-185.md

```diff
@@ -11,42 +12,35 @@ tags:
 👋 Emberistas! 🐹
 
 <SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
+Ember course on Educative 🆕,
+Using Ember with XState ❎,
 
 ---
 
-## [1. Section title in sentence case 🐹](section-url)
+<img alt="Ember on Educative" src="/images/educative.avif" />
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<add the contributor in the post in format "FirstName LastName (@githubUserName)" linked to their GitHub account>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+## [Developing Robust Ember.js Applications course on Educative 🆕](https://www.educative.io/courses/robust-ember-js-applications)
 
-<add your name to author list, top and bottom>
-<add short title to "SOME-INTRO-HERE">
+Based on the popular book, [Rock and Roll with Ember.js](https://balinterdi.com/rock-and-roll-with-emberjs/), Educative has a new (paid) course, [Developing Robust Ember.js Applications](https://www.educative.io/courses/robust-ember-js-applications) by [Balint Erdi (@balinterdi)](https://github.com/balinterdi). Educative is a growing online platform to level up your coding skills. Their teaching philosophy revolves around **no more passive learning**. Instead, the platform offers interactive in-browser environments to keep you engaged and test your progress as you go.
```

Comment:
I think the last 3 sentences regarding Educative may sound like an advertisement for the Educative platform (instead of an advertisement for the new Ember course).

Can we shorten them into 1 sentence (maybe)? To make the post longer, we could mention that Balint helped out with creating quizzes and challenges (source: marketing email from rarwe).

Reply:
@ijlee2 thanks for the review! made a tweak, lmk what you think

it seems like the PR view got a bit messed up, easier to see on ... > View file on the upper RHS